packages/react-sdk/src/user-props/use-user-props.ts

```diff
@@ -15,10 +16,15 @@ export const useUserProps = (instanceId: Instance["id"]) => {
   const propsData = allUserProps[instanceId];
   const props = useMemo(() => {
     if (propsData == null) return {};
-    return propsData.props.reduce((props, { prop, value }) => {
-      props[prop] = value;
-      return props;
-    }, {} as UserProps);
+    const result: UserProps = {};
+
+    for (const userProp of propsData.props) {
+      if ("value" in userProp) {
+        result[userProp.prop] = userProp.value;
+      }
+    }
+
+    return result;
```

Comment:
Probably a proper way to fix `useUserProps` is to make it support props that refer to an asset too, rather than ignoring them

Reply:
We did that with props transform then decided to move that logic into the component. As the image has better knowledge of what it needs from assets like our Image uses 2 asset props.